action.yml

```diff
@@ -153,4 +153,7 @@ runs:
           diff -B west.yml tmp.yml | patch west.yml - || true
           git add west.yml
           git commit --amend --no-edit
+          git fetch origin
+          git merge-tree --write-tree --name-only origin/main auto-manifest-${{ github.event.repository.name }}-${{ github.event.pull_request.number }}
+          if (( $? == 1)) ; then git rebase origin/main -X theirs ; fi
```

Comment:
Here we are always rebasing on `origin/main`. Does this action run on release branches? Is there an easy way to get the target branch for the PR?

Reply:
Good point! I believe this is another addition we should look into.

Reply:
I am fine with merging as-is, this is an improvement on the existing workflow.

Reply:
FYI: action only works for default branch currently. So release branches are not working. If release branches are named 1:1 between sub and manifest repos then I think it might be possible to add support for release branches in future.